packages/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -98,4 +98,5 @@ async function diffFolders (folderA: string, folderB: string) {
     .replace(new RegExp(`(a|b)${escapeStringRegexp(`/${normalizePath(folderBN)}/`)}`, 'g'), '$1/')
     .replace(new RegExp(escapeStringRegexp(`${folderAN}/`), 'g'), '')
     .replace(new RegExp(escapeStringRegexp(`${folderBN}/`), 'g'), '')
+    .replace(/.*No newline at end of file.*$/, '')
```

Comment:
If there is a "No newline at end of file" string in the code, will it be removed as well?

Reply:
Thanks a lot, you are right, we should only remove  the `/ No newlint at end of file` at the end of file.

Reply:
Can you add a unit test for this?

Reply:
Okay, there is just one case that needs to be added to the test ~